### ISSUE TITLE
NameError: name '_python_view_image_mod' is not defined

### ISSUE THREAD

**Issue by scowalt** [elazarcoh/simply-view-image-for-python-debugging#56]
I'm running VSCode inside of a docker container. I noticed that this extension doesn't work!

Here's my log (with some parts redacted): https://gist.github.com/scowalt/013583f667a8b9be44a0dfe964cd8fd4

**Comment by scowalt** [comment]
Here's [another log](https://gist.github.com/scowalt/81b72c2065709377fbc3f3f49c07a0ee). I've made sure this one included an `"event": "stopped"` event 

I've had to remove chunks of the log to avoid source leaks. I noticed that there are some errors:

```
Error finding viewables from expression `<file source code>`. Error: Syntax Error
```

**Comment by elazarcoh** [comment]
it is possible that the issue is as here: https://github.com/elazarcoh/simply-view-image-for-python-debugging/issues/46#issuecomment-1471510064_
TLDR install matplotlib and see if the issue persists.

**Comment by elazarcoh** [comment]
Yeah, but it's more of a bug, as matplotlib lib is not needed for the extension functionality.
dup of #54
